dlt-connector/src/proto/3_3/TransactionBody.ts

```diff
@@ -0,0 +1,66 @@
+import { Field, Message, OneOf } from '@apollo/protobufjs'
+
+import { CrossGroupType } from '@/graphql/enum/CrossGroupType'
+
+import { Timestamp } from './Timestamp'
+import { GradidoTransfer } from './GradidoTransfer'
+import { GradidoCreation } from './GradidoCreation'
+import { GradidoDeferredTransfer } from './GradidoDeferredTransfer'
+import { GroupFriendsUpdate } from './GroupFriendsUpdate'
+import { RegisterAddress } from './RegisterAddress'
+import { TransactionDraft } from '@/graphql/input/TransactionDraft'
+import { determineCrossGroupType, determineOtherGroup } from '@/controller/TransactionBody'
+
+// https://www.npmjs.com/package/@apollo/protobufjs
+// eslint-disable-next-line no-use-before-define
+export class TransactionBody extends Message<TransactionBody> {
+  public constructor(transaction: TransactionDraft) {
+    const type = determineCrossGroupType(transaction)
+    super({
+      memo: 'Not implemented yet',
+      createdAt: new Timestamp(new Date(transaction.createdAt)),
+      versionNumber: '3.3',
```

Comment:
what's the background of this versionNumber?
Is it a protobuf-version or a gradido-version?
Why is this versionNumber only part of the `TransactionBody` and not in the other classes of the directory `prot/3_3` too?

Reply:
It is the Gradido Protocol Version. Basically the version of the proto format definitions. 
Every Transaction has a TransactionBody. I don't see any usage to put the version in every class, because in the end TransactionBody is the main part of every GradidoTransaction and always there if it was send over to another server.  
Remember this image?
![class-diagram](https://github.com/gradido/gradido/blob/master/docu/Concepts/Snippets/images/classes.png)
Replace GradidoBlock with GradidoConfirmedTransaction.
I choose a new name for it the better understandable what it is. 
GradidoConfirmedTransaction is the only other with version number. I have some scenarios in my head where it come in handy.